index.html

```diff
@@ -0,0 +1,92 @@
+                        </svg>
+                        <svg width="16" height="12" viewBox="0 0 16 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fill-rule="evenodd" clip-rule="evenodd" d="M8.32997 2.60789C10.5459 2.60799 12.677 3.45943 14.283 4.98623C14.4039 5.1041 14.5972 5.10261 14.7163 4.98289L15.8723 3.81623C15.9326 3.7555 15.9662 3.67325 15.9657 3.58767C15.9652 3.50209 15.9306 3.42024 15.8696 3.36023C11.6545 -0.67932 5.00473 -0.67932 0.789632 3.36023C0.728573 3.4202 0.693929 3.50202 0.693366 3.5876C0.692803 3.67318 0.726368 3.75546 0.786632 3.81623L1.94297 4.98289C2.06199 5.1028 2.25544 5.10428 2.3763 4.98623C3.98245 3.45933 6.11385 2.60789 8.32997 2.60789ZM8.32997 6.40356C9.54745 6.40349 10.7215 6.85602 11.624 7.67323C11.746 7.78921 11.9383 7.78669 12.0573 7.66756L13.212 6.50089C13.2728 6.4397 13.3065 6.35668 13.3056 6.27042C13.3048 6.18415 13.2693 6.10184 13.2073 6.04189C10.4591 3.48551 6.20316 3.48551 3.45497 6.04189C3.39289 6.10184 3.35747 6.18419 3.35665 6.27049C3.35584 6.35678 3.3897 6.43979 3.45063 6.50089L4.60497 7.66756C4.72395 7.78669 4.91624 7.78921 5.0383 7.67323C5.94017 6.85656 7.11328 6.40406 8.32997 6.40356ZM10.643 8.95738C10.6447 9.04389 10.6107 9.12729 10.549 9.18789L8.55163 11.2036C8.49308 11.2628 8.41326 11.2961 8.32997 11.2961C8.24667 11.2961 8.16685 11.2628 8.1083 11.2036L6.11063 9.18789C6.04892 9.12724 6.01497 9.04381 6.0168 8.95731C6.01863 8.8708 6.05608 8.78888 6.1203 8.73089C7.39587 7.65201 9.26406 7.65201 10.5396 8.73089C10.6038 8.78893 10.6412 8.87087 10.643 8.95738Z" fill="white"/>
+                            </svg>
+                            <svg width="25" height="12" viewBox="0 0 25 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                <rect opacity="0.35" x="0.5" y="0.833252" width="21" height="10.3333" rx="2.16667" stroke="white"/>
+                                <path opacity="0.4" d="M23 4V8C23.8047 7.66122 24.328 6.87313 24.328 6C24.328 5.12687 23.8047 4.33878 23 4Z" fill="white"/>
+                                <rect x="2" y="2.33325" width="18" height="7.33333" rx="1.33333" fill="white"/>
+                            </svg>
+                                
```

Comment:
[OPTIONAL] I highly suggest that remove those icons because they aren't a part of the design requirements